### ISSUE TITLE
Durations in slowlog should use a simpler format

### ISSUE THREAD

**Issue by alex-lx** [pingcap/tidb#9014]
## Feature Request

**Is your feature request related to a problem? Please describe:**

We're using aliyun's log service to collect the slowlog, and found it's hard to parse the duration as a number. 
**Describe the feature you'd like:**

The slowlog print duration in a fixed format, eg 1.4s, 0.01s, 10000s
**Describe alternatives you've considered:**

Preprocess the slowlog using an external script
**Teachability, Documentation, Adoption, Migration Strategy:**

Easy to parse the slow in a log collector

**Comment by shenli** [comment]
@alex-lx Good suggestion.

**Comment by alex-lx** [comment]
@shenli I created a PL to resolve this, would you like to take a look?
